CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+## Code of conduct
+
+The team members and the project's community adopts an inclusive Code of Conduct that you can read in the [CODE_OF_CONDUCT_EN.md](CODE_OF_CONDUCT_EN.md) file.
+
+## Report an issue
+
+The prefered way to report any bug is [opening an issue in the project's Github repo](https://github.com/AjuntamentdeBarcelona/decidim-barcelona/issues/new).
+
+For more informal communication, contact team members via twitter
+
+## Resolve an issue
+
+Admins tag issues using two label related with collaboration availability:
+
+* `PRs-welcome`: [issues labeled with PRs-welcome](https://github.com/AjuntamentdeBarcelona/decidim-barcelona/labels/PRs-welcome) are well defined features ready to be implemented by whoever wants to do it.
+
+* `Not-ready`: with this label admins mark features or changes that are not well defined yet or subject to an internal decision. Is not a good idea to start implementation of these issues.
```

Comment:
This is not exactly how we'll be organizing ourselves and it mainly inherits from consul. Should we just copy it for now and modify it later? (famous last words)